src/vs/editor/contrib/suggest/suggestModel.ts

```diff
@@ -97,17 +97,25 @@ export const enum State {
 	Auto = 2
 }
 
-function shouldPreventQuickSuggest(contextKeyService: IContextKeyService, configurationService: IConfigurationService): boolean {
+function getDefault<T>(value: T | undefined, defaultValue: T): T {
+	return value === undefined ? defaultValue : value;
+}
```

Comment:
There is the new `??` operator for these kind of util functions...

Reply:
Ah, good to know. Usually, I try to avoid such operators, as I'm always unsure how it handles `0` and `false`. I guess that's exactly the difference between `??` and `||` though.

Reply:
Yeah, that's why `??` was added: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator